test/markup/asciidoc/default.txt

```diff
@@ -5,15 +5,15 @@ Author Name, <author@domain.foo>
 you can write text http://example.com[with links], optionally
 using an explicit link:http://example.com[link prefix].
 
-* single quotes around a phrase place 'emphasis'
+* single quotes around a phrase does not place 'emphasis'
```

Comment:
AsciiDoc Python and Asciidoctor are both implementations of the AsciiDoc syntax.
Historically, http://asciidoc.org was managed by the AsciiDoc Python project but AsciiDoc is now a registered "trademark" by Eclipse on the behalf of the AsciiDoc WG.
As a result, http://asciidoc.org will be updated to clarify the situation.

AsciiDoc Python is now deprecated, you can read on http://asciidoc.org:

> Asciidoctor provides a modern, compliant, and substantially faster implementation of the AsciiDoc processor written in Ruby. This implementation can also be run on the JVM (with AsciidoctorJ) or using JavaScript (with Asciidoctor.js). The Asciidoctor project now maintains the official definition of the AsciiDoc syntax.

I think this is confusing because AsciiDoc (Python) is named AsciiDoc but AsciiDoc is a syntax not a specific processor.

Likewise, Asciidoctor is not a syntax, it's a processor. Having said that, and until the AsciiDoc WG produces a specification, it's where the official definition of the AsciiDoc syntax is maintained.

> Does this change everything?

No, we are all good!
We are implementing a syntax highlighter for AsciiDoc (the syntax).

Reply:
As the lead of the Asciidoctor implementation and the chair of the AsciiDoc Working Group, I back up this statement by @Mogztter. The language is AsciiDoc and the asciidoc.org website is currently in the process of being transitioned to reflect that fact. The content that currently lives there is being phased out (and moved).

Reply:
@mojavelinux  So you're the lead on the Ruby implementation and on the `asciidoc` Python team as wel? :)

Reply:
@joshgoebel That's correct. For AsciiDoc.py, I'm really more than just a maintainer. I'm also the co-administrator with Lex, a duty we took on when Stuart left the project.

Reply:
Cool.  I'm not familiar with the term "co-administator", but I think I get the gist.  Here I typically say I'm the current "maintainer" (historically maintainers change every few years, churn) and that we have other core team members.  We have no co-administrators I don't think. :)

Reply:
I think you understood my point.